contribute/labels-of-issues.md

```diff
@@ -0,0 +1,116 @@
+# List of labels for issues and PRs on OpenEBS Project
+
+This document provides a list of labels and their purpose. These labels can be attached to issues and PRs. The aim behind having such list of labels is, we can sort and distinguish issues. This list will help new community members to dive-in OpenEBS if they wish to contribute to project or investigate the problem.
+
+Use case:
+> I am a beginner, I want to contribute to OpenEBS project and I am not sure from where to begin?
+
+Answer:
+> Start looking into the issues tagged with labels such as `help/small` , `size/XS`, `kind/unit-test` etc. Solve such issues and raise PR.
+
+---
+
+## Kind
+
+Labels prefixed with *kind* indicate the type of issue and PR. Suppose, you want to report a bug then you can tag your issue with `kind/bug` label.
+
+| Label | Description |
+|---|---|
+| kind/api  | Issue/PR related to API  |
+| kind/bug |  If there’s an error, failure in workflow, fault etc.|
+| kind/feature  | New / existing / required feature or enhancement |
+| kind/improve | Needs improvement |
+| kind/e2e-test | Need/Add end to end test |
+| kind/unit-test | Need/Add Unit test|
+| kind/security  | Security-related concerns  |
+| kind/testing  |  Needs/Adds tests |
+| kind/good-first  |  beginner friendly |
+
+---
+
+## Help
+
+Labels prefixed with *help* implies that issue needs some help from community/users. Further, you can adjust the level of help required, by specifying small, medium, large etc.
+
+| Label | Description |
+|---|---|
+| help/small  | Need small help on docs (beginner level)|
+| help/medium | Need help to fix the code ( Intermediate level) |
+| help/large | Need help on design changes and code review (Expert level) |
+
+---
+
+## Area
+
+Labels prefixed with *area* indicates that which area issue or PR belong. Here, area implies component of the OpenEBS project. It will be easier to sort an issue based on the area of user's interest.
+
+| Label | Description |
+|---|---|
+| area/networking  | Networking Specific  |
+| area/k8s | Kubernetes specific   |
+| area/ci  | Continuous integration specific  |
+| area/high-availability  | High availability  |
+| area/storage | Storage specific  |
+| area/machine-learning  |  Machine learning specific |
+
+---
+
+## Size
+
+Labels prefixed with *size* defines the size of change done in particular PR.
+
+| Label | Description |
+|---|---|
+| size/L | Size of change is large/7 commits probably |
+| size/M | Size of change is medium/5 commits probably |
+| size/S | Size of change is small/3 commits probably |
+| size/XL | Size of change is extra large/10 commits probably |
+| size/XS | Size of change is extra small/1 commit probably |
+| size/XXL | Size of change is very large/15 commits probably |
```

Comment:
update with the org size definitions please

Reply:
Done.